assets/queries/ansible/gcp/cluster_labels_disabled/query.rego

```diff
@@ -18,7 +18,7 @@ CxPolicy[result] {
 		"resourceName": task.name,
 		"searchKey": sprintf("name={{%s}}.{{%s}}", [task.name, modules[m]]),
 		"issueType": "MissingAttribute",
-		"keyExpectedValue": sprintf("%s is defined and not null", [modules[m]]),
+		"keyExpectedValue": sprintf("%s should be defined and not null", [modules[m]]),
 		"keyActualValue": sprintf("%s is undefined and null", [modules[m]]),
```

Comment:
Should we also change the KeyActualValue from "undefined **and** null" to "undefined **or** null "?
See assets/queries/ansible/gcp/cluster_master_authentication_disabled/query.rego as a reference.

Reply:
@kaplanlior you are more than right, but I decided to go over all the keyExpectedValue fields first and I'll go over all the keyActualValue fields in another PR if it's ok bu you.

Reply:
Sure, no problem with splitting that to another PR. Just want to make sure this doesn't get forgotten.